content/docs/engine-dev/tools/o3de-cli/project-export/high-level-overview.md

```diff
@@ -0,0 +1,100 @@
+---
+linkTitle: Overview
+title: Project Export Overview
+description: High Level Developer Overview for the Project Export command.
+---
+
+## Motivation
+
+Before the 23.10 release of O3DE, exporting game projects for sharing was a very difficult task for new users. There was a lot of cognitive burden to keep track of the tools required to process the game, requiring interaction with multiple independent parts (i.e. CMake, AssetProcessor, AssetBundler, etc.). For experienced users, there was also a lack of unified automation for the most common tasks.
+
+This caused the export experience to be highly error prone. 
+
+To put this in perspective, back in Q4 2022, an engineer took 1 week to manually package the Newspaper Delivery Game demo project. The same process with the Project Export command takes 2-15 minutes.
+
+The Project Export command also follows the spirit of modularity found in other parts of the O3DE ecosystem, as diverse needs grow overtime in the open source ecosystem.
+
+## Command Setup
+The `export-project` command has the following setup:
+
+![Project Export CLI setup](/images/engine-dev/o3de-cli/project-export/project-export-cli-setup.png)
+
+The central driver of the command is the Export Script. In O3DE, the Export Script is simply any python script that the user wishes to run using the O3DE Python CLI.
+
+The way this works is that O3DE CLI (`o3de.bat` on Windows or `o3de.sh` on Linux) [invokes](https://github.com/o3de/o3de/blob/development/scripts/o3de.py#L107-L108) the `export-project` [sub-command argument](https://github.com/o3de/o3de/blob/development/scripts/o3de.py#L79) via a specified [entry-hook](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/export_project.py#L484). This system heavily relies on the `argparse` standard library to configure the expected function for the parsed command in a modular fashion.
+
+Note that `export-project` only partially parses arguments. This is because the remaining arguments are fed through to the main Export Script when it runs, which will have its own argument parsing logic.
+
+## Execution Path
+The entry function is called [`_run_export_script`](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/export_project.py#L367), and is structured to start the setup of the export process in the following layers:
+
+##### Layer 1: **`_run_export_script`**  
```

Comment:
For each layer title, maybe consider using "###".

Reply:
That was what I originally wanted to do, but for each section heading that had an embedded link, markdown caused it to become transparent unless you moused over it. This was the only header style for the layer title that did not cause titles with links to disappear.